pipelines/h3c/rtp/v1/task_scripts/do_SSINS.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+# import common functions
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file. Here we define their positions,
+# which must be consisten wtih the config.
```

Comment:
A couple typos in this line

Reply:
Thanks. Just fixed this in a commit a few seconds ago.